malwarefront/src/components/Tag.js

```diff
@@ -24,7 +24,7 @@ export class Tag extends Component {
                         this.props.searchable ?
                             <Link to={makeSearchLink("tag", this.props.tag, false, this.props.searchEndpoint)}
                                   style={{all: "unset", cursor: "pointer"}}
-                                  onClick={ev => this.props.tagClick(ev, this.props.tag)}>{this.props.tag}</Link>
+                                  onClick={ev => this.props.tagClick && this.props.tagClick(ev, this.props.tag)}>{this.props.tag}</Link>
```

Comment:
This seems a bit hacky :thinking:
Is there a reason why `tagClick` might not be defined there?

Reply:
It's optional, sometimes we have only direct link to search.

Actually it is a bit hacky because Tag component should be slightly redesigned. Both action and link props should be generalized, this component should just render tag badge with optional remove/unfilter buttons instead of doing lots of things like evaluating actual search link.